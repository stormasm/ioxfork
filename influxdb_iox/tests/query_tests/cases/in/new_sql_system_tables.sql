-- IOX_SETUP: TwoMeasurementsManyFieldsTwoChunks

-- validate we have access to information schema for listing system tables
-- IOX_COMPARE: sorted
SELECT * from information_schema.tables where table_schema = 'system';

-- IOX_COMPARE: sorted
-- Note the issue_time changes, so we can't display it directly
-- Instead check that it is reasonable and non null
SELECT issue_time <= now(), query_type, query_text, success FROM system.queries;
