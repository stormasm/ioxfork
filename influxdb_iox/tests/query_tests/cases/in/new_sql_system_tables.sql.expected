-- Test Setup: TwoMeasurementsManyFieldsTwoChunks
-- SQL: SELECT * from information_schema.tables where table_schema = 'system';
-- Results After Sorting
+---------------+--------------+------------+------------+
| table_catalog | table_schema | table_name | table_type |
+---------------+--------------+------------+------------+
| public        | system       | queries    | BASE TABLE |
+---------------+--------------+------------+------------+
-- SQL: SELECT issue_time <= now(), query_type, query_text, success FROM system.queries;
-- Results After Sorting
+------------------------------------+------------+----------------------------------------------------------------------------------+---------+
| system.queries.issue_time <= now() | query_type | query_text                                                                       | success |
+------------------------------------+------------+----------------------------------------------------------------------------------+---------+
| true                               | sql        | SELECT * from information_schema.tables where table_schema = 'system';           | true    |
| true                               | sql        | SELECT 1;                                                                        | true    |
| true                               | sql        | SELECT issue_time <= now(), query_type, query_text, success FROM system.queries; | false   |
+------------------------------------+------------+----------------------------------------------------------------------------------+---------+